libraries/ESP8266SSDP/ESP8266SSDP.cpp

```diff
@@ -71,7 +71,7 @@ static const char _ssdp_packet_template[] PROGMEM =
   "%s" // _ssdp_response_template / _ssdp_notify_template
   "CACHE-CONTROL: max-age=%u\r\n" // SSDP_INTERVAL
   "SERVER: Arduino/1.0 UPNP/1.1 %s/%s\r\n" // _modelName, _modelNumber
-  "USN: uuid:%s\r\n" // _uuid
+  "USN: %s\r\n" // _uuid
```

Comment:
why is the prefix moved out of the template? it increases permanent heap use by 10 bytes due to the increased _uuid size, and I don't see a good reason for it. I prefer to keep the prefix here.

Reply:
check this review :
https://github.com/esp8266/Arduino/pull/5750#pullrequestreview-205353824